src/shared/performance/transactions.ts

```diff
@@ -0,0 +1,24 @@
+import * as Sentry from '@sentry/react-native'
+import '@sentry/tracing'
+import { Span, Transaction } from '@sentry/types'
+
+import { eventMonitoring } from 'libs/monitoring'
+
+const transactionsRef: { [key: string]: { transaction: Transaction; span: Span } | undefined } = {}
+
+export function startTransaction(name: string) {
+  if (transactionsRef[name]) {
+    finishTransaction(name)
+  }
+  const transaction = eventMonitoring.startTransaction({ name })
+  Sentry.getCurrentHub()?.configureScope((scope) => scope.setSpan(transaction))
+  const span = transaction?.startChild({ op: 'performance' })
```

Comment:
Si on se fie au typage de la lib :
```suggestion
  Sentry.getCurrentHub().configureScope((scope) => scope.setSpan(transaction))
  const span = transaction.startChild({ op: 'performance' })
```
T'as eu des problèmes en faisant comme ça ?